camunda-sdk-java/java-client-operate/README.md

```diff
@@ -0,0 +1,69 @@
+# Java Operate client
+
+## Usage in your project
+
+This client is part of the Spring Zeebe project. If you intend to use it outside of this project, you can add a dependency to io.camunda.spring:java-client-operate
+```xml
+<dependency>
+	<groupId>io.camunda.spring</groupId>
+	<artifactId>java-client-operate</artifactId>
+	<version>8.3.4</version>
+</dependency>
+```
+
+## Build the client
+
+### SaaS Authentication
+
+```java
+  JwtConfig jwtConfig = new JwtConfig();
+  jwtConfig.addProduct(Product.OPERATE, new JwtCredential(clientId, clientSecret));
+  targetOperateUrl = "https://" + region + ".operate.camunda.io/" + clusterId;
+  auth = SaaSAuthentication.builder().jwtConfig(jwtConfig).build();
+
+  client = CamundaOperateClient.builder()
+              .operateUrl(targetOperateUrl)
+              .authentication(auth)
+              .setup()
+              .build();
+```
+
+### SelfManaged Authentication
+
+```java
+  JwtConfig jwtConfig = new JwtConfig();
+  jwtConfig.addProduct(Product.OPERATE, new JwtCredential(clientId, clientSecret));
+  auth = SelfManagedAuthentication.builder().jwtConfig(jwtConfig).keycloakUrl(keycloakUrl).build();
+
+  client = CamundaOperateClient.builder()
+              .operateUrl(operateUrl)
+              .authentication(auth)
+              .setup()
+              .build();
+```
```

Comment:
I just recently updated and fixed the SimpleAuthentication. Do you want to include it as well? Or I can include it after?

Reply:
Yes, I'll let you do. I'm wondering if we should document it in the java-common and just link it from the client to not repeat ourselves when other clients will be integrated